action.yml

```diff
@@ -22,6 +22,18 @@ inputs:
     description: 'What flake8 version to use, if none is installed (default is latest)'
     required: false
     default: ''
+  plugins:
+    description: 'List of plugins to install before running, This is passed directly to pip install'
+    required: false
+    default: ''
+  error_classes:
+    description: 'List of flake8 error classes to classify as Error.'
+    required: false
+    default: 'E,F'
+  warning_classes:
+    description: 'List of flake8 error classes to classify as Warning.'
+    required: false
+    default: 'W'
```

Comment:
I wonder if we shouldn't add `B`, `H`, .. as defaults, just to make it easier for people. Or maybe we should make this `[^EF]`, or: anything not in the error-class. To make sure every entry is at least annotated? Not sure :)

Reply:
> Or maybe we should make this [^EF], or: anything not in the error-class.

I do not dislike that, i actually was gonna comment about a posible "bug" with having error_classes and warning_classes in that if one of those is left empty it will break the pipeline.

I'll code in the negation and you tell me what you think :)